public/docs/dart/latest/tutorial/toh-pt5.jade

```diff
@@ -48,648 +48,658 @@ p Run the #[+liveExampleLink2('', 'toh-5')] for this part.
 :marked
-    Set the `AppComponent`’s `styleUrls` property to this CSS file.
+  Set the `AppComponent`’s `styleUrls` property to this CSS file.
 +makeExample('toh-5/dart/lib/app_component.dart','style-urls', 'lib/app_component.dart (styleUrls)')(format=".")
 :marked
-    ### Global application styles
-    When we add styles to a component, we're keeping everything a component needs
-    &mdash; HTML, the CSS, the code &mdash; together in one convenient place.
-    It's pretty easy to package it all up and re-use the component somewhere else.
+  ### Global application styles
+  When we add styles to a component, we're keeping everything a component needs
+  &mdash; HTML, the CSS, the code &mdash; together in one convenient place.
+  It's pretty easy to package it all up and re-use the component somewhere else.
+
+  We can also create styles at the *application level* outside of any component.
 
-    We can also create styles at the *application level* outside of any component.
+  Our designers provided some basic styles to apply to elements across the entire app.
+  These correspond to the full set of master styles that we 
+  introduced earlier (see 
+  [QuickStart, "Add some style"](../quickstart.html#!#add-some-style)).
+  Here is an excerpt.
 
-    Our designers provided some basic styles to apply to elements across the entire app.
-    Add the following to a new file named `styles.css` in the root folder.
-+makeExample('toh-5/ts/styles.1.css', '', 'styles.css (App Styles)')(format=".")
++makeExample('toh-5/ts/styles.1.css', 'toh-excerpt', 'styles.css (app styles excerpt)')(format=".")
+
+- var styles_css = 'https://raw.githubusercontent.com/angular/angular.io/master/public/docs/_examples/styles.css'
 
 :marked
-    Reference this stylesheet within the `index.html` in the traditional manner.
+  Add a new file named `styles.css` in the root folder, if there isn't one already.
+  Ensure that it contains the [master styles given here](!{styles_css}).
+
+  Also ensure this stylesheet is referenced in the traditional manner within `index.html`.
 +makeExample('toh-5/ts/index.html','css', 'index.html (link ref)')(format=".")
 :marked
-    Look at the app now. Our dashboard, heroes, and navigation links are styling!
+  Look at the app now. Our dashboard, heroes, and navigation links are styling!
 
 figure.image-display
-    img(src='/resources/images/devguide/toh/dashboard-top-heroes.png' alt="View navigations")
+  img(src='/resources/images/devguide/toh/dashboard-top-heroes.png' alt="View navigations")
 
 .l-main-section
 :marked
-    ## Application structure and code
-
-    Review the sample source code [in the live example for this chapter](/resources/live-examples/toh-5/ts/plnkr.html).
-    Verify that we have the following structure:
+  ## Application structure and code
+p.
+  Review the sample source code in the #[+liveExampleLink2('', 'toh-5')] for this part.
```

Comment:
Why change chapter to part? I thought they were consistently using "chapter" for each page?